packages/manager/modules/pci/src/projects/project/project.component.js

```diff
@@ -4,6 +4,7 @@ import template from './project.html';
 export default {
   bindings: {
     projectId: '<',
+    serviceId: '<',
```

Comment:
We already have projectId. Can't we use same ?

Reply:
in this context service and project are two diffrent entities.